fiftyone/utils/sam.py

```
@@ -647,270 +662,317 @@ class SegmentAnythingModel(fout.TorchSamplesMixin, fout.TorchImageModel):
+        # Adapted from segment-anything.modeling.sam.SAM.forward.
+        input_images = torch.cat(
+            [self._model.preprocess(img) for img in images], dim=0
+        )
+        image_embeddings = self._model.image_encoder(input_images)
+
+        point_coords = imgs.get("point_coords")
+        point_labels = imgs.get("point_labels")
+        boxes = imgs.get("boxes")
+        mask_inputs = imgs.get("mask_inputs")  # Not used currently
+        multimask_output = (
+            True if (boxes is None and point_coords is not None) else False
```

Ruff (0.15.7)

[warning] Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)